src/java.base/share/classes/java/time/LocalDate.java

```diff
@@ -2147,11 +2147,23 @@ public int hashCode() {
      */
     @Override
     public String toString() {
+        var buf = new StringBuilder(10);
+        formatTo(buf);
+        return buf.toString();
+    }
+
+
+    /**
+     *  Outputs this date into {@code StringBuilder}, such as {@code 2007-12-03}.
```

Comment:
You shouldn't copy the docs for `toString` as that is not we should be careful about using this `formatTo`. You should write something helpful like:
```java
/**
 * Prints the toString result to the given buf, avoiding extra string allocations.
 * Requires extra capacity of 10 to avoid StringBuilder reallocation.
 */
```